src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -106,7 +100,8 @@ def build(self):
         self.gas_constant = Param(within=NonNegativeReals,
                                   mutable=False,
                                   default=0.008314,
-                                  doc='Gas Constant [MJ/(kgmol.K)]')
+#                                  units=pyunits.MJ/pyunits.kmol/pyunits.K,
+                                  doc='Gas Constant [MJ/(kmol.K)]')
```

Comment:
Note that this should be replaced by the value in the IDAES constants module: `from idaes.core.util.constants import Constants`. It will need to have a unit conversion applied to it wherever it is used however.

Reply:
Would this need to be updated in the state block file as well?

Reply:
Yes - in fact there should be no need for this in the parameter block. You should just use it where needed in the state block.

Reply:
Makes sense, thank you @andrewlee94